### ISSUE TITLE
Fine-tune on office-31

### ISSUE THREAD

**Issue by Marsrocky** [jindongwang/transferlearning#38]
As Jindong points out, I also cannot reimplement the source-only result on office-31 by fine-tuning. Is anyone successful?

**Comment by jindongwang** [comment]
I suppose this will always be a mystery...

**Comment by Marsrocky** [comment]
> I suppose this will always be a mystery...

I guess the reason lies in the library we use. If we use ResNet-50 on Pytorch, the accuracy is similar. But for AlexNet, the authors adopted Caffe.

**Comment by jindongwang** [comment]
Yes...
